tools/clang/test/CodeGenSPIRV/type.constant-buffer.multiple-dimensions.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %dxc -T ps_6_0 -E main -fspv-target-env=universal1.5
+
+struct Foo {
+  float4 a;
+  int2 b;
+};
+
+// CHECK: %_ptr_Uniform__arr__arr_type_ConstantBuffer_Foo_uint_3_uint_2 = OpTypePointer Uniform %_arr__arr_type_ConstantBuffer_Foo_uint_3_uint_2
```

Comment:
Relying on exact names may be quite brittle, I don't think we guarantee the naming scheme will remain as-is. I think it would be more robust to match each Op that creates this final type step by step.